Source/WebKit/Shared/API/Cocoa/WKRemoteObjectCoder.mm

```diff
@@ -783,6 +801,11 @@ static void checkIfClassIsAllowed(WKRemoteObjectDecoder *decoder, Class objectCl
     if (allowedClasses->contains((__bridge CFTypeRef)objectClass))
         return;
 
+    if (alwaysAllowedClasses().contains((__bridge CFTypeRef)objectClass))
+        return;
+
+    RELEASE_LOG_FAULT(IPC, "Unexpected class %s", NSStringFromClass(objectClass).UTF8String);
```

Comment:
If you always want this logged, create a separate channel other than `IPC`.  (I would recommend doing this.)